mackup/utils.py

```diff
@@ -16,6 +16,9 @@
 # If True, the user wants to say "yes" to everything.
 FORCE_YES = False
 
+# Flag that control if mackup can be run as root
+ROOT = False
```

Comment:
I'd make this more explicit. `CAN_RUN_AS_ROOT`?

Reply:
sure